apps/bfd-data-npi/README.md

```diff
@@ -0,0 +1,6 @@
+BFD NPI Org Utility
+=======================================
+
+## Description
+
+This jar is used to download NPI Org names from the NPI org website.  The npi org names will be used for the API for npi lookups.
```

Comment:
I suggest some changes here:

BFD NPI to Organization Name Lookup
======

This maven project produces a jar file that provides functionality to retrieve organization names given an NPI as input. This lookup is backed by a data file stored in the jar files as a resource.

The data file is constructed from \<fill in details here about where we get the file\>.

The data file is constructed by \<fill in details here about the original file format and how we filter certain rows and extract certain columns ; I think you can copy most of it from the story and just make the formatting markdown compatible\>